tests/e2e-pw/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const shouldRunSubscriptionsTests =
+	process.env.SKIP_WC_SUBSCRIPTIONS_TESTS !== '1';
+
+export const products = {
```

Comment:
I'm unsure about the names of those constants. I wanted to use `SHOULD_RUN_SUBSCRIPTIONS_TESTS`, but since ESLint flagged it, I opted to keep both in lower camel case for now.